From dd68d8efe2c4a5c3f04317d71e359b2d579fcf59 Mon Sep 17 00:00:00 2001
From: chrome-release-bot <chrome-release-bot@chromium.org>
Date: Fri, 26 Jun 2020 15:05:21 +0000
Subject: [PATCH 05/12] Incrementing VERSION to 83.0.4103.120

TBR=benmason@chromium.org

Change-Id: I2de6fbef6b1d5ba6d9bbd5296e9706d7ca3194e2
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2269505
Reviewed-by: Chrome Release Bot (LUCI) <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/4103@{#728}
Cr-Branched-From: 8ad47e8d21f6866e4a37f47d83a860d41debf514-refs/heads/master@{#756066}
---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index 7c0499cafac..f7310a92cf3 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=83
 MINOR=0
 BUILD=4103
-PATCH=119
+PATCH=120
-- 
2.28.0

